src/__mock__/Homepage.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
- [ ] You did a great job. However, there are some changes that need to be done. You wrote, mock, and united test. But, you forgot to add integration tests. Kindly write integration tests as well. 

> Testing requirements
> 
> Create unit tests for pure functions in your Redux code.
> Create integration tests for your application using the React Testing Library.
> You may need to mock the access to the API, so that your tests don't send actual requests.
> You may need to mock the connection to the Redux Store.

Reply:
well resolved! ✔️